engine/src/main/java/org/hibernate/search/util/logging/impl/Log.java

```diff
@@ -1004,4 +1004,12 @@
 
 	@Message(id = 328, value = "Cannot create context for class: %1$s" )
 	SearchException cannotCreateBridgeDefinedField(@FormatWith(ClassFormatter.class) Class<?> backend, @Cause Exception e);
+
+	@Message(id = 329, value = "Property " + Environment.ANALYZER_DEFINITION_PROVIDER + "set to value '%1$s' is invalid."
```

Comment:
- you need a space just before `set to value`.
- maybe better to quote the Environment.ANALYZER_DEFINITION_PROVIDER with single quotes?
- I'd not request "must not throw any exception", seems redundant. Also a bit scary.. will the data center blow up if an OOM happens in there?
- optional alternative: may be more useful to remind that a public, no-argument constructor is expected.